common/djangoapps/enrollment/views.py

```diff
@@ -4,6 +4,7 @@
 
 """
 import logging
+import json
```

Comment:
As new imports are added, they should be placed in alphabetical order; please move this up one line.